### ISSUE TITLE
Selection of reactions not accessible via keyboard navigation.

### ISSUE THREAD

**Issue by BurntimeX** [WoltLab/WCF#3703]
ref https://community.woltlab.com/thread/286250-accessibility-funktionen-%C3%BCber-tastatur-nicht-erreichbar/
